src/Aspire.Hosting.Python/PythonAppResourceBuilderExtensions.cs

```diff
@@ -53,24 +53,24 @@ public static class PythonAppResourceBuilderExtensions
     /// var builder = DistributedApplication.CreateBuilder(args);
     ///
     /// builder.AddPythonApp("python-project", "PythonProject", "main.py");
-    /// 
-    /// builder.Build().Run(); 
+    ///
+    /// builder.Build().Run();
     /// </code>
     /// </example>
     public static IResourceBuilder<PythonAppResource> AddPythonApp(
-        this IDistributedApplicationBuilder builder, string name, string projectDirectory, string scriptPath, params string[] scriptArgs)
+        this IDistributedApplicationBuilder builder, string name, string appDirectory, string scriptPath, params string[] scriptArgs)
     {
         ArgumentNullException.ThrowIfNull(builder);
 
-        return builder.AddPythonApp(name, projectDirectory, scriptPath, ".venv", scriptArgs);
+        return builder.AddPythonApp(name, appDirectory, scriptPath, ".venv", scriptArgs);
     }
 
     /// <summary>
     /// Adds a python application with a virtual environment to the application model.
     /// </summary>
     /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/> to add the resource to.</param>
     /// <param name="name">The name of the resource.</param>
-    /// <param name="projectDirectory">The path to the directory containing the python project files.</param>
+    /// <param name="appDirectory">The path to the directory containing the python app files.</param>
     /// <param name="scriptPath">The path to the script relative to the project directory to run.</param>
```

Comment:
Is the `project directory` being referred to here the app host project directory or the Python app directory? I think it's the latter, which if so this should change to read something like "The path to the script to run, relative to the app directory."